src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/consumer/wa/TaskManagerGetTaskConsumerTest.java

```diff
@@ -245,6 +246,17 @@ private DslPart createResponseForGetTaskForWa() {
                         )
                         .stringType("next_hearing_id", "nextHearingId")
                         .datetime("next_hearing_date", "yyyy-MM-dd'T'HH:mm:ssZ")
+                        .object("permissions", (value) -> {
+                            value
+                                .unorderedArray("values", (p) -> p
+                                    .stringValue(PermissionTypes.READ.value())
+                                    .stringValue(PermissionTypes.COMPLETE.value())
+                                    .stringValue(PermissionTypes.CLAIM.value())
+                                    .stringValue(PermissionTypes.UNCLAIM.value())
+                                    .stringValue(PermissionTypes.UNASSIGN_CLAIM.value())
+                                    .stringValue(PermissionTypes.CANCEL.value())
+                                    .stringValue(PermissionTypes.EXECUTE.value()));
```

Comment:
this appears to be for senior-tribunal-caseworker. Should this include UnassignAssign & UnclaimAssign?
Should Cancel be on this particular list?

Reply:
From base branch 1528